lib/os.js

```diff
@@ -33,7 +33,7 @@ const {
 
 const { safeGetenv } = internalBinding('credentials');
 const constants = internalBinding('constants').os;
-const isWindows = process.platform === 'win32';
+const { isWindows } = require('internal/util');
```

Comment:
Wouldn't adding a `require` statement negatively affect loading?

IIUC the less `require` statements, the better.